Web_Development/Javascript/Topics/6.Variables/README.md

```diff
@@ -0,0 +1,164 @@
+## Javascript Variables
+
+In this article you are going to learn everything about **Javascript Variables** from scratch. 
+
+Variables are "containers" for storing information JavaScript variables are used to hold values or
+expressions. A variable can have a short name, like x, or a more descriptive name, like carname.
+
+
+
+ ## Rules for declaring Javascript Variable name:
+There are some rules while declaring a JavaScript variable (also known as identifiers).
+
+   1. Name must start with a letter (a to z or A to Z), underscore( _ ), or dollar( $ ) sign.
+   2. After first letter we can use digits (0 to 9), for example value1.
+   3. JavaScript variables are case sensitive, for example x and X are different variables.
+
+
+
+Creating variables in JavaScript is most often referred to as "declaring" variables. We declare
+JavaScript variables with the var keyword let's have a look on the syntax.
+
+## Syntax:-
+```
+var x;
+var name;
+you can also assign values to the variables when you declare them:
+var x=5;
+var name="Ram";
+
+```
```

Comment:
Please use \`\`\`js instead of just \`\`\`. This will enable syntax highlighting,